jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaLayerConfigurations.java

```diff
@@ -97,6 +98,11 @@ public Builder setExtraFiles(List<Path> extraFiles) {
       return this;
     }
 
+    public Builder setExplodedWar(List<Path> explodedWar) {
```

Comment:
Hmm, didn't we want to split the exploded war into the classes, resources, and dependencies layers?

Reply:
Yeah, that is true. The single layer is for the first iteration.

Reply:
Let's add a TODO to remove this? (and elsewhere where we want to move the exploded war files into the classes/resources/dependencies? like `JavaDockerContextGenerator`)